Patches/Vanilla Factions Expanded - Ancients/PawnKinds_Spacer.xml

```diff
@@ -29,25 +29,23 @@
           </value>
         </li>
 
-        <li Class="PatchOperationAddModExtension">
-          <xpath>/Defs/PawnKindDef[defName="VFEA_AncientSoldierTwoAbilities" or defName="VFEA_PlayerAncientSoldierOneAbility"]</xpath>
-          <value>
-            <li Class="CombatExtended.LoadoutPropertiesExtension">
```

Comment:
Why'd you remove their ammo???

Reply:
the patch was borked and had duplicate entries for the VFEA_PlayerAncientSoldierOneAbility, the VFEA_AncientSoldierTwoAbilities was added to the first patch's address